test/Token.js

```diff
@@ -6,6 +6,11 @@
 // We import Chai to use its asserting functions here.
 const { expect } = require("chai");
 
+// We use `loadFixture` as an alternative to Mocha's `beforeEach` because
+// `loadFixture` takes advantage of Hardhat Network's snapshot functionality in
+// order to avoid redeploying the contract for every test run.
+const { loadFixture } = require("@nomicfoundation/hardhat-network-helpers");
```

Comment:
```suggestion
// We use `loadFixture` to share common setups (or fixtures) between tests.
// Using this simplifies your tests and makes them run faster, by taking advantage
// or Hardhat Network's snapshot functionality.
const { loadFixture } = require("@nomicfoundation/hardhat-network-helpers");
```

I think it's better not to assume that the reader is familiar with `beforeEach`.